.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
Was this meant to be pushed ? Otherwise, it should be added in the `.gitignore`

Reply:
Good point! Looks like MS visual code has added these auxiliary files. Will remove them! Thanks.